### DISCUSSION TITLE
how to route mdx page?

### DISCUSSION THREAD

**Discussion by dotku** [vercel/next.js#70457] [App Router] [answered]
eg I have path with `src/app/business/business-devlopment-letter.mdx` but I can't access it directly by `http://localhost:3000//business/business-devlopment-letter` any suggestion?

And the OpenAI seems doesn't know about it either:
https://chatgpt.com/share/66f3f0c6-95b0-8001-96a0-0803a63b9f4a

**Comment by Juneezee** [comment, marked as answer]
See https://nextjs.org/docs/app/building-your-application/configuring/mdx#using-file-based-routing

You filename needs to be `page.mdx`, i.e. `src/app/business/business-devlopment-letter/page.mdx`

**Reply by dotku** [reply]
Thanks, @Juneezee, it is working!!!

**Reply by Juneezee** [reply]
@dotku You're welcome :smiley:
